src/vsr/replica.zig

```diff
@@ -5271,15 +5356,19 @@ pub fn ReplicaType(
         /// `view_durable` and `log_view_durable` will update asynchronously, when their respective
         /// updates are durable.
         fn view_durable_update(self: *Self) void {
-            assert(self.status == .normal or self.status == .view_change);
+            assert(self.status == .normal or self.status == .view_change or
+                (self.status == .recovering and self.solo()));
```

Comment:
To allow for the "view-change" (just incrementing its view numbers, not a real view change) that a `replica_count=1` replica does during open.